src/libmongoc/tests/test-mongoc-topology.c

```diff
@@ -2202,6 +2203,94 @@ test_slow_server_pooled (void)
    mock_server_destroy (primary);
    checks_cleanup (&checks);
 }
+
+static void
+_test_ismaster_versioned_api (bool pooled)
+{
+   mock_server_t *server;
+   mongoc_uri_t *uri;
+   mongoc_client_pool_t *pool;
+   mongoc_client_t *client;
+   char *ismaster;
+   future_t *future;
+   request_t *request;
+   bson_error_t error;
+   mongoc_server_api_version_t version;
+   mongoc_server_api_t *api;
+
+   server = mock_server_new ();
+   mock_server_run (server);
+   uri = mongoc_uri_copy (mock_server_get_uri (server));
+
+   mongoc_server_api_version_from_string ("1", &version);
```

Comment:
Nit, because this returns `false` on an error you could `BSON_ASSERT` this works.

Reply:
Not a nit ;)

Done.